engine/src/main/scala/cromwell/engine/io/nio/NioFlow.scala

```diff
@@ -101,6 +101,7 @@ class NioFlow(parallelism: Int,
       read.file.limitFileContent(read.options.maxBytes, read.options.failOnOverflow),
       StandardCharsets.UTF_8
     )
+      .replaceAll("\\r\\n", "\\\n")
```

Comment:
Minor comment (don't worry if this isn't possible): Is it possible to do the replace _during_ the stream rather than as a post-processing step on the downloaded string?